neural_compressor/torch/quantization/config.py

```diff
@@ -69,8 +69,11 @@ class RTNConfig(BaseConfig):
         "use_sym",
         "use_full_range",
         "use_mse_search",
-        "use_layer_wise",
         "export_compressed_model",
+        # layer wise params
+        "use_layer_wise",
+        "model_path",
+        # double quant
```

Comment:
The arguments have been modified, please update the usage in the examples [examples](https://github.com/intel/neural-compressor/tree/master/examples/3.x_api/pytorch/nlp/huggingface_models/language-modeling/quantization/llm) accordingly. It is ok to do it in a separate PR. 
We would better create a JIRA to track it.

Reply:
Got it, thanks.